src/azure-cli/azure/cli/command_modules/network/_params.py

```diff
@@ -138,6 +143,7 @@ def load_arguments(self, _):
         c.argument('cert_password', help='The certificate password')
         c.argument('http_settings_port', help='The HTTP settings port.')
         c.argument('servers', ag_servers_type)
+        c.argument('key_vault_secret_id', help="Secret Id of (base-64 encoded unencrypted pfx) 'Secret' or 'Certificate' object stored in Azure KeyVault. You need enable soft delete for keyvault to use this feature.", is_preview=True)
```

Comment:
What kind of message does server return when no identity assigned or no access policy set?

Reply:
identity should be set. The server will check the existence of the identity